batchflow/tests/notebooks/torch_test.ipynb

```diff
@@ -1459,7 +1459,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3",
```

Comment:
Something gone wrong with this file update?

Reply:
no idea really